practices/reduce-coupling-between-abstractions.md

```diff
@@ -86,3 +94,24 @@ Clear abstractions and minimal dependencies support better documentation and col
 * Have we identified the most problematic areas of coupling in our codebase?
 * What small, incremental changes can we make to start reducing coupling in these areas?
 * How do we ensure system stability while refactoring to reduce coupling?
+
+
+### [Do a Spike](/practices/do-a-spike.md)
+
+<ed> This may be a silly question, but does your target reader know what a spike is? I'm assuming they do and we don't need to define this practice, but just want to double-check. </ed>
+
+
+- **Refactor**: Set some time aside to refactor a key component or set of components to reduce coupling. Present your findings to the team to see if committing those changes or making additional changes have a good potential return on investment.
+- **Audit Your Dependencies**: Use a dependency analysis tool to visualize the relationships between modules and components, and to identify highly coupled areas. Discuss why these dependencies exist. <ed> Is there a specific tool(s) you'd recommend? </ed>
```

Comment:
Is there a specific tool(s) you'd recommend?

Reply:
Sorry, I didn't see this until just now. You can link to this guy's website who does a decent job at listing what's out there.

Reply:
https://markgacoka.medium.com/how-to-visualize-your-codebase-7c4c4d948141

Reply:
Done!